src/pfs/pfs.proto

```diff
@@ -140,6 +141,17 @@ message Trigger {
   string cron_spec = 6;
 }
 
+message BranchPropagationSpec {
+  // TODO: Picker?
+  Branch branch = 1;
+  PropagationSpec propagation_spec = 2; 
+}
+
+message PropagationSpec {
```

Comment:
Is the propagation spec going to be reused for other resources in PFS? If not, I think you can just inline the fields.

Reply:
Yeah, in PPS, which I just pushed up.